src/test/test-namespace.c

```diff
@@ -208,6 +208,7 @@ TEST(protect_kernel_logs) {
                                     NULL,
                                     NULL,
                                     NULL,
```

Comment:
An assertion to extension directory is added. So this triggers the assertion, no?

Reply:
oops, yes indeed - removed the assert, it's redundant anyway, there are other asserts before it is actually used (which won't happen in those tests as the options triggering its use are unset)

Reply:
and of course I botched the check, as `append_extensions()` is always called (with empty args) - moved the assert in there below the check for the optional args